portfolio-website/script.js

```diff
@@ -1,23 +1,79 @@
+// navBar
 const navBarList = [
-    "Ingredients", "Steps to Make", "Links to Other Wonton Soup Recipes", "About Me"
+    "About Me", "Projects", "Contact"
 ]
 
 for (let i = 0; i < navBarList.length; i++) {
     const label = document.createElement('a');
```

Comment:
generally if we create an reference a specific html tag, we can reference it by tag name
so in this case

const link = document.createElement('a');
addColorOnEvent(link, 'mouseover', 'black');
addColorOnEvent(link, 'mouseout', 'black');

helps w readability